Objects/longobject.c

```diff
@@ -47,6 +47,11 @@ is_small_int(long long ival)
 {
     return -NSMALLNEGINTS <= ival && ival < NSMALLPOSINTS;
 }
+static inline int
+is_small_uint(uintmax_t ival)
```

Comment:
Ah, good thought re `uintmax_t`!

It'd be a nice small fix to have `is_small_int` take an `intmax_t`, too. I had forgotten that type existed when I wrote it.